nx-X11/programs/Xserver/include/os.h

```diff
@@ -460,6 +460,11 @@ extern _X_EXPORT size_t
 strlcat(char *dst, const char *src, size_t siz);
 #endif
 
+#ifndef HAVE_TIMINGSAFE_MEMCMP
```

Comment:
We probably also need changes to Imake, given that for instance `OpenBSD` does have a `timingsafe_memcmp` function.

See `nx-X11/config/cf/Imake.tmpl` and `nx-X11/programs/Xserver/os/Imakefile` and look for `strlcat` for how that was done.